enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/state/machines/token/ReplicatedTokenRequestSerializer.java

```diff
@@ -45,6 +46,7 @@
 {
     private ReplicatedTokenRequestSerializer()
     {
+        throw new AssertionError();
```

Comment:
Since we're adding an assertion here, perhaps a message describing what the assertion is (class should not be instantiated) is in order.